src/Microsoft.AspNetCore.Authorization/AuthorizationHandlerContext.cs

```diff
@@ -33,14 +39,34 @@ public class AuthorizationHandlerContext
             _pendingRequirements = new HashSet<IAuthorizationRequirement>(requirements);
         }
 
+        /// <summary>
+        /// The collection of all the <see cref="IAuthorizationRequirement"/> for the current authorization action.
+        /// </summary>
         public IEnumerable<IAuthorizationRequirement> Requirements { get; }
+
+        /// <summary>
+        /// The <see cref="ClaimsPrincipal"/> representing the current user.
+        /// </summary>
         public ClaimsPrincipal User { get; }
+
+        /// <summary>
+        /// The optional resource to evaluate the <see cref="AuthorizationHandlerContext.Requirements"/> against.
+        /// </summary>
         public object Resource { get; }
 
+        /// <summary>
+        /// Gets the requirements that have not yet been succeeded.
```

Comment:
@blowdart is off site so I'll make the edits to get it in today, I went with "not yet been marked as succeeded."

FYI the matching method is Succeed() that needs to be called for the given requirement..